File: ControlAssuranceAPI/Models/GoAssignment.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GoAssignment
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? GoElementId { get; set; }

    public int? UserId { get; set; }

    public virtual GoElement? GoElement { get; set; }

    public virtual User? User { get; set; }
}

File: control-assurance-sp/src/types/GoAssignment.ts
import { IEntity } from "./Entity";
import { IGoElement } from "./GoElement";
import { IUser } from "./User";

export interface IGoAssignment extends IEntity {
    GoElementId?: number;
    UserId?: number;
    GoElement?: IGoElement;
    User?: IUser;
}

export class GoAssignment implements IGoAssignment {
    public ID = 0;
    public Title = null;
    public GoElementId = null;
    public UserId = null;
}
